frontend/src/hocs/withExpandable/withExpandable.tsx

```diff
@@ -48,12 +48,12 @@ function withExpandable(
       ...componentProps
     } = props
 
-    const [expanded, toggleExpanded] = useState<boolean>(initialExpanded)
+    const [expanded, setExpanded] = useState<boolean>(initialExpanded)
     useEffect(() => {
-      toggleExpanded(initialExpanded)
-    }, [initialExpanded])
+      setExpanded(initialExpanded)
+    }, [label, initialExpanded])
```

Comment:
I would add a comment on why the label matters here.

Reply:
done 👍